spec/lib/citation_string_processor_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require './lib/emory/citation_string_processor'
+
+RSpec.describe CitationStringProcessor, type: :helper do
+  describe '#append_string_with_comma' do
+    it 'appends a comma if field present' do
+      expect(helper.append_string_with_comma(["blah"])).to eq("blah, ")
+    end
+
+    it 'returns nil if field nil' do
```

Comment:
It might be useful to test this method and `#append_string_with_period` with `nil` as well

Reply:
Will take care of this in next push.